netaddiction_theme_rewrite/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Netaddiction
+
+{
+    'name': 'Netaddiction Theme Rewrite',
+    'category': 'eCommerce',
+    'version': '1.0',
```

Comment:
@FabioNetaddiction Puoi usare quest'altro schema di versionamento? 
```suggestion
    'version': '14.0.1.0.0',
```